### DISCUSSION TITLE
Cannot open the port

### DISCUSSION THREAD

**Discussion by HaoFmr** [dicebear/dicebear#288] [Q&A] [answered]
Hi there,

When I try to host API with docker, I cannot open the port: http://0.0.0.0:3000. Do I need to do some setting?

**Comment by FlorianKoerner** [comment, marked as answer]
The example in the documentation was missing the port forwarding setting. I updated the documentation: https://dicebear.com/guides/host-the-http-api-yourself#with-docker
